x-pack/plugins/upgrade_assistant/public/application/components/tabs/checkup/deprecations/reindex/flyout/warnings_step.tsx

```diff
@@ -131,35 +128,6 @@ export const WarningsFlyoutStep: React.FunctionComponent<WarningsConfirmationFly
 
         <EuiSpacer />
 
-        {warnings.includes(ReindexWarning.allField) && (
-          <WarningCheckbox
-            checkedIds={checkedIds}
-            onChange={onChange}
-            warning={ReindexWarning.allField}
-            label={
-              <FormattedMessage
-                id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.warningsStep.allFieldWarningTitle"
-                defaultMessage="{allField} will be removed"
-                values={{
-                  allField: <EuiCode>_all</EuiCode>,
-                }}
-              />
-            }
-            description={
-              <FormattedMessage
-                id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.warningsStep.allFieldWarningDetail"
-                defaultMessage="The {allField} meta field is no longer supported in 7.0. Reindexing removes
-                      the {allField} field in the new index. Ensure that no application code or scripts reply on
-                      this field."
-                values={{
-                  allField: <EuiCode>_all</EuiCode>,
-                }}
-              />
-            }
-            documentationUrl={`${esDocBasePath}/6.0/breaking_60_mappings_changes.html#_the_literal__all_literal_meta_field_is_now_disabled_by_default`}
-          />
-        )}
-
         {warnings.includes(ReindexWarning.apmReindex) && (
```

Comment:
note: the `apm` warning still remains for now; I need to sync with the observability team next week to better understand the requirements for the 8.0 upgrade.